projects/dsp-ui/src/lib/viewer/values/date-value/date-value.component.ts

```diff
@@ -45,8 +45,6 @@ export class DateValueComponent extends BaseValueComponent implements OnInit, On
 
     matcher = new ValueErrorStateMatcher();
 
-    dateEditable = true;
-
     constructor(@Inject(FormBuilder) private _fb: FormBuilder,
                 private _valueService: ValueService) {
```

Comment:
I don't think you need the `ValueService` here anymore

Reply:
done in https://github.com/dasch-swiss/dsp-ui-lib/pull/298/commits/7a2d826e2bde11fc6dc1d86196109d1c2fcb7c70